functions/Set-DbaPowerPlan.ps1

```diff
@@ -83,160 +68,68 @@ function Set-DbaPowerPlan {
 
     #>
     [CmdletBinding(SupportsShouldProcess)]
-    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseOutputTypeCorrectly", "", Justification = "PSSA Rule Ignored by BOH")]
     param (
         [parameter(ValueFromPipeline)]
         [DbaInstance[]]$ComputerName,
         [PSCredential]$Credential,
-        [string]$PowerPlan = 'High Performance',
+        [Alias("PowerPlan")]
         [string]$CustomPowerPlan,
```

Comment:
"Custom" has a specific meaning in Power Plan world. High Performance is not custom, it is standard. Not sure I see the value in changing this parameter. In addition, it is not mandatory and I can't find where it is set. Does it ever get set?

Reply:
The magic is in Test-DbaPowerPlan. There is the best practice powerplan hard coded. But if you use $CustomPowerPlan, then this would be considered best practive. Test-DbaPowerPlan uses Get-DbaPowerPlan to get all power plans and returns all the information needed (names and IDs) to set the best practice power plan.

Reply:
I do love that functionality! I'm still confused as to why `Custom` is in there. Perhaps we just swap and make CustomPowerPlan the alias?

Reply:
Yes, no problem. But we should have the same name in Test- and Set-.

Reply:
Ok, done that.